### PR TITLE
Cleanup Spark images.

### PR DESCRIPTION
**Pull request by ValentinMouret**
@ValentinMouret please add comment with repository:tag on top of each new docker file

### CODE REVIEW

**Comment by bruwozniak** [comment]
@ValentinMouret was there not a pull request template available when you opened it? I thought I set it successfully

**Comment by ValentinMouret** [comment]
> @ValentinMouret was there not a pull request template available when you opened it? I thought I set it successfully



> @ValentinMouret was there not a pull request template available when you opened it? I thought I set it successfully

I think I created my PR just before you committed. Another cause may be that I created the PR from the CLI and not from the GitHub UI

**Comment by bruwozniak** [comment]
@ValentinMouret yeah it's because of CLI

File: spark/data-science-base/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM pubnative/pyspark-ci:base-e849b23
```
Comment (ztanis) [inline spark/data-science-base/Dockerfile]: please add `# pubnative/pyspark-ci:version-tag`

Comment (ztanis) [reply spark/data-science-base/Dockerfile]: @bruwozniak @miry we end up with `# pubnative/pyspark-ci:data-science-base-${COMMIT}` comment

**Pull request merged**